services/transaction_body.proto

```diff
@@ -186,6 +188,11 @@ message TransactionBody {
      */
     CryptoUpdateTransactionBody cryptoUpdateAccount = 15;
 
+    /**
+     * An Ethereum encoded transaction.
+     */
+    EthereumTransactionBody ethereumTransaction = 51;
```

Comment:
Making ethereum transaction schedulable was never in scope to my knowledge, there is no ethereum-land equivalent.  If you are scheduling calls you are already in HAPI and should be using ContractCallTransaction. Especially when you consider the interactions with the etherumNonce, it's a sure path to broken transactions.

Reply:
There's a separate scheduling whitelist property that ultimately controls what is allowed and only a few transaction bodies that are in `SchedulableTransactionBody` are currently listed. So I think it's appropriate to populate this in case we want to in the future make it schedulable, but please check with Michael.

Reply:
Actually read your entire comment. If you're sure it can never logically make sense to schedule it, then yeah we wouldn't want this added.

Reply:
I'd rather have to add it to scheduled at a later release than add it and have a long list of "this is how it breaks" scenarios.  The list is very long, and the remediation of "just use ContractCallTransaction" is so very simple we don't need that level of complexity in the code.